.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
+  workflow_dispatch:
+
+
+jobs:
+  build_package_deploy:
+    name: Deploy Module
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PowerShell module cache
+        id: cacher
+        uses: actions/cache@v3
+        with:
+          path: "~/.local/share/powershell/Modules"
+          key: ${{ runner.os }}-PSModules
+      - name: Setup
+        shell: pwsh
+        run: |
+          ./Tools/setup.ps1
+          Invoke-Build -Task ShowInfo
+      - name: Build
+        shell: pwsh
+        run: |
+          Invoke-Build -Task Clean, Build
```

Comment:
do we need this?
can't we download the artifact from `Upload Artifact` from `.github/workflows/build_and_test.yml`?